src/components/Button/BaseButton.tsx

```diff
@@ -80,8 +81,13 @@ export const BaseButton: FC<InternalButtonProps> = React.forwardRef(
             { [styles.buttonSize2]: size === ButtonSize.Medium },
             { [styles.buttonSize3]: size === ButtonSize.Small },
             { [styles.pillShape]: shape === ButtonShape.Pill },
+            {
+                [styles.roundShape]:
+                    shape === ButtonShape.Round && !split && !textExists,
```

Comment:
@dkilgore-eightfold Looks like two-state button doesn't call baseButton at all and everything is handled there on its own. So here I'm only handling "split". Verified that when set to round, split button still show rectangular